docs/_data/navigation.yml

```diff
@@ -6,7 +6,5 @@
   link: /add-um-users.html
 - name: Add Non-UM Users
   link: /add-non-um-users.html
-- name: Format Canvas Gradebook
-  link: /gradebook-canvas.html
```

Comment:
This changes looks good. I just realized that somebody could access the URL by directly typing it. /gradebook-canvas.html. Like how I did it https://praveshkunwar.github.io/canvas-course-manager-next/gradebook-canvas.html

This is highly unlikely, But can you rename the file gradebook-canvas-retire.html (Things might be cache, so wait a bit and see)

Reply:
Yep, renamed it.